### ISSUE TITLE
Ubuntu 14.10 is not supported

### ISSUE THREAD

**Issue by tjyang** [saltstack/salt-bootstrap#972]
### Description of Issue/Question

I have a few ubuntu 14.10 and they can't be bootstrapped by ./bootstrap_salt.sh
### Setup

(Please provide relevant configs (Be sure to remove sensitive info).)
### Steps to Reproduce Issue

<snipped>
     W: Failed to fetch https://repo.saltstack.com/apt/ubuntu/14.10/amd64/latest/dists/trusty/main/binary-amd64/Packages  HttpError404

```
              W: Failed to fetch https://repo.saltstack.com/apt/ubuntu/14.10/amd64/latest/dists/trusty/main/binary-i386/Packages  HttpError404
```

<snipped>
### Versions and Systems

./bootstrap_salt.sh -v
./bootstrap_salt.sh -- Version 2016.08.16

**Comment by rallytime** [comment]
Hmm...Yeah we don't package 14.10 as it is not part of our [supported distros](http://saltstack.com/wp-content/uploads/2016/08/SaltStack-Supported-Operating-Systems.pdf), but we do list `14.x` on the [README](https://github.com/saltstack/salt-bootstrap/blob/stable/README.rst#ubuntu-and-derivatives) of this repo. 

Thanks for filing this. I wonder if we could update the bootstrap script to try to install/run the 14.04 packages on a 14.10 machine, like we're doing for some of the SLES 11 installations...

**Comment by rallytime** [comment]
@tjyang I discussed this with @thatch45 and we have decided that we won't be fixing this. 14.10 is EOL from Ubuntu and has been for over a year. Our policy is to only support LTS releases from Ubuntu, and 14.10 does not fall under that policy. I have updated the bootstrap script in #1002 to only support 12.04, 14.04, and 16.04 (with best-effort support for 16.10) as discussed in #976.

With those changes, if you run the script on a non-supported ubuntu release, you'll get an error message saying it is not supported and it will exit. I have also updated the README to make this more clear.
